link/src/main/java/com/stripe/android/link/ui/LinkAppBarState.kt

```diff
@@ -47,9 +31,8 @@ internal fun rememberLinkAppBarState(
             } else {
                 R.drawable.stripe_link_back
             },
-            showHeader = showHeader,
             showOverflowMenu = showOverflowMenu,
-            email = email?.takeUnless { it.isBlank() || hideEmail },
+            email = email?.takeUnless { it.isBlank() },
```

Comment:
Is this behavior change? Do we have a test for this?

Reply:
This will effectively only be used in the verification dialog. The behavior change is removing the context of a bunch of other screens.

Will likely end up removing even more from here in a follow up.